tests/integration/s2n_dynamic_record_size_test.py

```diff
@@ -200,54 +199,40 @@ def test(host, port, test_ciphers, threshold):
 
     return failed
 
-def analyze_latency_dump(array):
-    failed = 0
+
+def analyze_tcp_dump(array, threshold):
```

Comment:
Can you add a comment describing this function.

Reply:
Done :) [#1e2620f](https://github.com/awslabs/s2n/pull/2292/commits/1e2620fc4a7e9085f5b52b9796bc2799ce3edc93)